src/index.html

```diff
@@ -1,13 +1,64 @@
+<!-- #region head -->
 <!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Akatab:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
     <link rel="stylesheet" href="./style.css">
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        class="preview"
+        src="images/imac.jpeg"
+        alt="imac"
```

Comment:
13 [CHECKLIST] - alt atribute should describe the image if the image contains information (better description you have - better for you :))

Reply:
<img width="1051" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/49764881/92bc2e93-915a-4897-8458-ec8812c43b9a">